src/crypto/openssl/openssl_crypto_accel.cc

```diff
@@ -13,8 +13,63 @@
  */
 
 #include "crypto/openssl/openssl_crypto_accel.h"
-#include <openssl/aes.h>
+#include <openssl/evp.h>
+#include <openssl/engine.h>
+#include "common/debug.h"
 
+// -----------------------------------------------------------------------------
```

Comment:
Do the comments here need to be removed?

Reply:
I prefer to keep them here, just like what other plugin codes do